compiler/rustc_symbol_mangling/src/typeid/typeid_itanium_cxx_abi.rs

```diff
@@ -745,6 +745,19 @@ fn encode_ty<'tcx>(
     typeid
 }
 
+/// Creates a trait object.
+#[inline]
+fn new_dynamic_trait<'tcx>(
```

Comment:
This helper is adding more code than it's eliding; please inline it.

Reply:
Done.